test/e2e/break_glass_credentials_test.go

```diff
@@ -0,0 +1,114 @@
+//go:build e2e
+// +build e2e
```

Comment:
This validation is great but we don't need to spin up a new HC for it. We can let this be a validation of the createCluster test or the buildin logic that validate any new HC used in test.